### ISSUE TITLE
Admin command to expire all auctions for player

### ISSUE THREAD

**Issue by srleojaco** [Crazy-Crew/CrazyAuctions#91]
A command like **/ca forceexpire [player]** that would disappear al items from auctions/sell menú for selected player.

**Comment by SrLeojaco** [comment]
@badbones69

**Comment by badbones69** [comment]
Thanks, ill add it to my list of suggestions as this makes sense. Unsure when ill have time to add this though.

**Comment by marvinej** [comment]
I agree with this. Some additional administrative commands to this plugin would be very helpful. Perhaps also a command to open, edit, and remove individual player auctions.

**Comment by RyderBelserion** [comment]
Added to project board.
